enclave/scripts/enclave.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+: '
+  In execution, this script is managed by PM2. On startup, yarn start is called.
+  In case the enclave crashes, yarn start:recover is called.
+'
+
+if [ -f "encryption_key.txt" ]
```

Comment:
was thinking we'd build this the server itself (typescript), but i don't see an obvious advantage of that solution vs your current 

so looks good to me